app/policies/patient_policy.rb

```diff
@@ -13,10 +13,10 @@ def create?
   end
 
   def update?
-    user.present?
+    user.present? && owner?
```

Comment:
Looks like we have multiple `user.present? && owner?`, any ideas about how to solve this?

Reply:
I'll change `owner?` method to extend this behavior. All owner method calls need to check if user is present too. b0457a4adda1a2067db56f4d4a7937a82eb9be28